From 15b370fea1e2b03f0f226fb8bcdbe6e41003a6a4 Mon Sep 17 00:00:00 2001
From: Tk-Glitch <ti3nou@gmail.com>
Date: Sun, 12 May 2019 12:54:53 +0200
Subject: No God, Please No! - staging
Unless it's used to build pure mainline or staging, it's not proper for issue reporting


diff --git a/dlls/kernel32/process.c b/dlls/kernel32/process.c
index acdefa28a1..45afdacc05 100644
--- a/dlls/kernel32/process.c
+++ b/dlls/kernel32/process.c
@@ -1266,11 +1266,11 @@ void WINAPI start_process( LPTHREAD_START_ROUTINE entry, PEB *peb )
     {
         if (CreateEventA(0, 0, 0, "__winestaging_warn_event") && GetLastError() != ERROR_ALREADY_EXISTS)
         {
-            FIXME_(winediag)("Wine Staging %s is a testing version containing experimental patches.\n", wine_get_version());
-            FIXME_(winediag)("Please mention your exact version when filing bug reports on winehq.org.\n");
+            FIXME_(winediag)("Wine TkG %s is a testing version containing experimental patches.\n", wine_get_version());
+            FIXME_(winediag)("Please don't report bugs about it on winehq.org and use https://github.com/Frogging-Family/wine-tkg-git/issues instead.\n");
         }
         else
-            WARN_(winediag)("Wine Staging %s is a testing version containing experimental patches.\n", wine_get_version());
+            WARN_(winediag)("Wine TkG %s is a testing version containing experimental patches.\n", wine_get_version());
 
 
         if (!CheckRemoteDebuggerPresent( GetCurrentProcess(), &being_debugged ))
